app/posts/how-to-use-cross-government-slack.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to use cross-government Slack
+description: Tips on how to join and make the most of the cross-government Slack workspace.
+date: 2023-06-27
+tags: post
+layout: post
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt: 
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It's a place to connect with other people working on digital services across government.
```

Comment:
(Me being overly finicky, part 1) Although I think quotes render correctly regardless of input text, think we use smart quotes in source copy (just checked and that’s mostly the case, will go and fix the ones in previous posts that were missed!)

This is the first of a few ‘straight’ quotes:

```suggestion
[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It’s a place to connect with other people working on digital services across government.
```